.github/actions/constellation_destroy/action.yml

```diff
@@ -51,13 +55,13 @@ runs:
         echo "::endgroup::"
 
     - name: Login to GCP (Cluster service account)
-      if: inputs.cloudProvider == 'gcp'
+      if: always() && inputs.cloudProvider == 'gcp'
```

Comment:
Nit: using [`continue_on_error`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) may be a bit more elegant.